packages/wallet-backend-update/tests/auth/controller.test.ts

```diff
@@ -0,0 +1,192 @@
+import { createContainer } from '@/createContainer'
+import { Bindings } from '@/app'
+import { env } from '@/config/env'
+import { Container } from '@/shared/container'
+import { createApp, TestApp } from '../app'
```

Comment:
How about add tests to tsconfig as well? So that this can be @/tests/app

Reply:
I like this approach and I'll look into it. Let's hope TS will not yell at us.